ethereum/setprotocol_v2/set_component_token_mapping.sql

```diff
@@ -0,0 +1,33 @@
+-- https://dune.xyz/queries/576574
+-- There are certain wrapped components of Set tokens that map closely 1:1 with their corresponding token
+-- but are not actively traded, since they're just individual tokens that represent claims on other tokens
+
+
+create table if not exists setprotocol_v2.set_component_token_mappings
+(
+  symbol varchar,
+  component_address bytea,
+  token_type varchar,
+  mapped_symbol varchar,
+  mapped_component_address bytea         
+);
+
+create index if not exists on setprotocol_v2.set_component_token_mappings(component_address, mapped_component_address);
```

Comment:
@jackiep00 i've created this table & inserted the rows, however, the indices were not created due to syntax error. refer to your other table script, where there is a name to be applied before the 'on' command. please fix that syntax and push a commit to this PR, then i'll apply them. i'll continue on without the index here